webapp/utils/websocket_client.jsx

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import WebSocketClient from 'mattermost/websocket_client.jsx';
+
+var WebClient = new WebSocketClient();
+export default WebClient;
```

Comment:
Why duplicate this here instead of importing mattermost/websocket_client directly? Also just an FYI, you can replace the last two lines with
```
export default new WebSocketClient();
```